src/oneNoteDataStructures/oneNoteItemUtils.ts

```diff
@@ -2,11 +2,8 @@ import { Notebook } from './notebook';
 import { OneNoteItem } from './oneNoteItem';
 import { Section } from './section';
 import { SectionGroup } from './sectionGroup';
-import { Polyfills } from '../polyfills';
```

Comment:
I want to drop the polyfill from teams and this one will not let us do that. I think the library should include this polyfill as it is pretty small

Reply:
This isn't supported on IE so I think this makes sense to include here.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find